docs/source/release_notes/release-notes.rst

```diff
@@ -42,6 +42,15 @@ develop
     *    - Type
          - Change
 
+    *    - |devbreak|
+         - The implementation of the TimeLock server has been separated into a new project, `timelock-impl`.
```

Comment:
We should mention that most users should not be directly affected, unless they have been depending on classes from within TimeLock.